source/ci_source/ci_source.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+export interface CISource {
+    /** hello */
+    env: any,
+    /** validates */
+    isCI: (env: any) => boolean,
+    /** PRs */
+    isPR: (env: any) => boolean
+}
```

Comment:
hrm, this interface isn't being conformed to yet - ideally I can write a test for this or something, but I don't know